tmail-backend/deployment-tests/common/src/main/java/com/linagora/tmail/deployment/JmapContract.java

```diff
@@ -79,7 +82,7 @@ default void mailboxGetTest() {
    }
 
    @Test
-   default void sessionRouteShouldReturnDelegatedAccounts() {
+   default void delegationShouldWork() {
```

Comment:
Not delegation specifically, the jmap routes need to work :)

Reply:
`delegationShouldWork` test in `JmapContract` => delegation with JMAP routes should work?
delegation with JMAP routes works -> no delegation should work anyway?

Reply:
delegation or not our jmap routes should work on deployment tests... it's not because this test breaks that it's necessary linked to delegation?